geoviews/plotting/bokeh/plot.py

```diff
@@ -70,6 +70,11 @@ def __init__(self, element, **params):
                     'and disable the show_grid option.'
                 )
 
+        self._unwrap_lons = False
+        if isinstance(self.geographic, _CylindricalProjection):
+            x1, x2 = element.range(0)
```

Comment:
I think element here is likely misnamed, we probably have to move this into some other method, e.g. `initialize_plot` and `update_frame`.